deploy-manage/remote-clusters/remote-clusters-migrate.md

```diff
@@ -29,7 +29,7 @@ If you run into any issues, refer to [Troubleshooting](remote-clusters-troublesh
 
 ## Prerequisites [remote-clusters-migration-prerequisites]
```

Comment:
does this page apply to anything other than self-managed?

Reply:
Content looks self-managed only. If you're commenting because of missing `applies` tags, that's because I'm waiting until we have a proper "self-managed" one.

Reply:
because of its location in the nav

<img width="240" alt="image" src="https://github.com/user-attachments/assets/0a2fde5d-ce83-4d8c-9a28-b73096e4e04e" />

Reply:
I believe this is the correct location as the instructions are currently centered on self-managed